packages/app/src/app/pages/Dashboard/Components/Sandbox/index.tsx

```diff
@@ -147,6 +149,7 @@ const GenericSandbox = ({ isScrolling, item, page }: GenericSandboxProps) => {
 
   const selected = selectedIds.includes(sandbox.id);
   const isDragging = isAnythingDragging && selected;
+  const isViewOnly = !hasActiveSubscription && sandbox.privacy !== 0;
```

Comment:
Privacy being an integer: 🥲

Reply:
Raised this with Julien, it's the same on the backend unfortunately. We can define an enum in the FE maybe to make it friendlier

Reply:
@julien-leclercq wants to change this to something else too :)